knowledge-base/customize-headers-pdf-radwordsprocessing.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Creating Headers with Left, Center, and Right Text/Image in Exported PDF Documents Using RadWordsProcessing
+description: Learn how to customize headers in PDF documents to have left, center, and right aligned text using RadWordsProcessing library.
+type: how-to
+page_title: How to Customize Headers in PDF Documents with RadWordsProcessing
+slug: customize-headers-pdf-radwordsprocessing
+tags: radwordsprocessing, document processing, header, alignment, pdf, convert html to pdf
+res_type: kb
+ticketid: 1665701
+---
+
+## Environment
+
+| Version | Product | Author | 
+| --- | --- | ---- | 
+| 2024.3.806| RadWordsProcessing |[Desislava Yordanova](https://www.telerik.com/blogs/author/desislava-yordanova)| 
+
+## Description
+When converting HTML documents to PDF format, it might be necessary to customize the document header to include left, center, and right aligned text and images. This article demonstrates how to generated a [RadFlowDocument]({%slug radwordsprocessing-model-radflowdocument%}) and divide a header into three sections and customize the font settings for each section respectively using the [RadWordsProcessing]({%slug radwordsprocessing-overview%}) library.
```

Comment:
Refactored: This article demonstrates how to generate a RadFlowDocument, divide a header into three sections, and customize the font settings for each section respectively using the RadWordsProcessing library.